.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve tephi!
+title: "[BUG]"
```

Comment:
@ESadek-MO Do you prefer to prefix titles with `[BUG]` et al instead of using `labels` ?

Reply:
I don't really mind either way, happy to lean towards your preference!
This was just suggested online, and I thought it could be a cool way to know at a glance, even if it's as an email notification.

Reply:
Happy to roll with that :+1: